.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+## PR Checklist
+Please check that your PR fulfills the following requirements:
+
+- [ ] The commit message follows [our guidelines](https://github.com/pesto-students/batch-10-tripen/wiki/Commit-message-guidelines)
+- [ ] Tests for the changes have been added (for bug fixes / features)
+- [ ] Docs have been added / updated (for bug fixes / features)
+- [ ] GIFs/Videos/Images added (compulsory for frontend changes)
+
+## PR Type
+What kind of change does this PR introduce? 
+
+<!-- Please check the one that applies to this PR using "x". -->
+- [ ] Bugfix
+- [ ] Feature
+= [ ] Tests changed / added / updated
+- [ ] Code style update (formatting, local variables)
+- [ ] Refactoring (no functional changes, no api changes)
+- [ ] Build related changes
+- [ ] CI related changes
+- [ ] Documentation content changes
+- [ ] Infrastructure changes
+- [ ] Dependency upgrade
+- [ ] Other... Please describe:
+
+## Issue Number: 
+#
```

Comment:
maybe add Resolves: # too here ?

Reply:
Isn't it the same thing? I can change `## Issue Number: ` to `# Resolves issue number: # ` if that sounds better?

Reply:
Resolves key word will auto close the issue , not sure if Resolves issue numbe will. Just double check once :)